src/query.rs

```diff
@@ -1394,8 +1433,46 @@ impl<'q, Q: Query> PreparedView<'q, Q> {
             .as_ref()
             .map(|fetch| fetch.get(meta.location.index as usize))
     }
+
+    /// Like `get_mut`, but allows checked simultaneous access to multiple entities
+    ///
+    /// For N > 3, the checking will clone the array and take O(N log N) time.
```

Comment:
If we're going to make one of multiple similar methods the canonical one and give it more detailed docs, we should link to it from the non-canonical methods (and probably omit the duplicated details entirely).

Reply:
Fixed as suggested.